src/routes.json

```diff
@@ -148,18 +148,16 @@
         "title": "Ideas"
     },
     {
-        "name": "jobs",
+        "name": "jobs-redirect",
         "pattern": "^/jobs/?$",
         "routeAlias": "/jobs(/moderator)?/?$",
```

Comment:
You can remove the `routeAlias` key, we no longer need since we need to remove the whole jobs-moderator block (see comment below)

Reply:
✔️ Done